dev/pages/Toggle.vue

```diff
@@ -50,13 +56,17 @@ watch(() => variants.active, changeToggleModel)
       <w-toggle :radio="active('Radio')" :checkbox="active('Checkbox')" invalid v-model="invalidToggleModel" label="A very INVALID toggly label" :toggles="toggles" />
     </token>
     <div>
-    <token :state="[variants, toggleModel]">
-      <w-toggle radio-button :equal-width="isJustified" v-model="multiToggleModel" label="A very toggly label" :toggles="toggles" />
-      <w-button class="mt-16" small utility @click="isJustified = !isJustified">{{ isJustified ? 'Unjustify' : 'Justify' }}</w-button>
+      <token :state="[variants, toggleModel]">
+        <w-toggle radio-button :equal-width="isJustified" v-model="multiToggleModel" label="A very toggly label" :toggles="toggles" />
+        <w-button class="mt-16" small utility @click="isJustified = !isJustified">{{ isJustified ? 'Unjustify' : 'Justify' }}</w-button>
+      </token>
+    </div>
+
+    <token>
+      <w-toggle indeterminate checkbox label="An indeterminate label" :toggles="indeterminateToggle" />
```

Comment:
Maybe it is just the examples that are "bad"... but clicking these indeterminate toggles renders them as both checked and indeterminate.
![image](https://github.com/warp-ds/vue/assets/7531505/a1ee0ff7-d1a6-4c47-8d7f-318ede06d687)

Reply:
Ah, of course, good catch! Indeterminate is normally set dynamically when there are multiple options, but not all are selected. Here I'd need to know if checkbox is checked, and if so, remove the `indeterminate` flag. I'm struggling a bit to pull it off in Vue though 😓

Reply:
Fixed in [9f8bafa](https://github.com/warp-ds/vue/pull/144/commits/9f8bafa5661dca82cad4db1e7b951d7168852e1e)